ideas/307-breaking-whiisper.md

```diff
@@ -0,0 +1,131 @@
+
+In an attempt to streamline and incentivize this process, as well as experiment with primitives of a DAO, we will be encapsulating this project with the SITG experiments, which are outlined in the above HackMD docs. 
+
+### Requirements & Dependencies
+- Whisper Simulator and metrics
+
+## [Milestones](#milestones)
+### Milestone 1: Simulation engine completion and metrics definition.
+Goal Date: 2018-
+Description: 
+* Completion of [whispervis](https://github.com/status-im/whispervis) and the required stats as laid out in Product Overview HackMD doc.
+* The simulation aspect is not as important as the ability to run multiple tests with different parameters changed in a quick manner
+
+Contributors: 
+* @divan ??
+* @adamb ??  
+* @arnetheduck ??
+
+### Milestone 2: Perform theoretical tests.
+Goal Date: 2018-
+Description:
+* Systematically perform and record all tests with regards to relevant parameter changes
+
+Contributors: 
+* @corpetty 
+
+### Milestone 3: Results aggregation and data analysis.
+Goal Date: 2018-
+Description:
+* Gather and scrub all data for uniformity
+* Plot relevant plots as per HackMD doc in Product Overview
+* Decide which results are best ported to experimental model
+  * practicality
+  * value of information for comparison
+* This should provide preliminary data on theoretical limit of naive whisper scaling
+  * Publish a blog about current progress, current insights, proposed next steps
+
+Contributors: 
+* @easye ??
+
+### Milestone 4: Beta fleet preparation.
+Goal Date: 2018-
+Description:
+* Complete description of cluster setup specifications
+* Make sure we are tracking the same metrics on the cluster as the theoretical model
+* Note typical usage / remove standard usage
+* Development of methodology for stress testing Beta fleet in the similar manner as Theoretical model. 
+
+Contributors: 
+* @jacob ??
+
+### Milestone 5: Perform beta fleet stress test.
+Goal Date: 2018-
+* Systematically apply methodology from previous milestone to actual experimental cluster
+* Collect and store relevent performance metrics
+* Publish a blog about initial experimental results and thoughts
+
+### Milestone 6: Comparison Analysis
+Goal Date: 2018-
+* Compare theoretical vs experimental tests data
+* Plot and create indicator of how accurate theory is to experiment
+* brainstorm on
+  * problems with theoretical model and experimental setup
+  * improvements with communication scaling and their implementation details with experimental/theoretical frameworks
+
+### Milestone 7: Paper write-up and publish
+Goal Date: 2018-
+* Write up the entire process / analysis / results / and conclusions for ourselves and the public. 
+* Portal for community feedback
+
+Contributors: 
+* Someone from peopleops for portal ??
+
+## Success Metrics
```

Comment:
Perhaps these two also acts as exit criteria?

Do you think 'implementation of scaling improvements' fit within scope of this idea?

Reply:
not initially.  Future work will dictate what we should focus on.  I think the paper on just the naive implementation is sufficient to start.